### PR TITLE
ref: Don't call beforeSend for internal minidump

### PR DESCRIPTION
**Pull request by HazAT**

### CODE REVIEW

File: src/main/backend.ts [hunk 1]
```diff
@@ -154,8 +147,17 @@ export class MainBackend extends BaseBackend<ElectronOptions> implements CommonB
...
-          captureEvent({ __INTERNAL_MINIDUMP: true });
+          getCurrentHub().captureEvent(
+            {
+              // @ts-ignore __INTERNAL_MINIDUMP is not assignable to event
+              __INTERNAL_MINIDUMP: true,
+            },
+            {
+              data: {
+                __sentry__: true,
+              },
```
Comment (HazAT) [inline src/main/backend.ts:158]: if an event has this in the hint, we will not invoke beforeSend.

Comment (jan-auer) [reply src/main/backend.ts:158]: Do we even need the `__INTERNAL_MINIDUMP` then?

Comment (HazAT) [reply src/main/backend.ts:158]: Yes we do, this `__sentry__` is used internally if our SDK captures something itself.
The `__INTERNAL_MINIDUMP`, while being a workaround is something else.

**Comment by vyunikov** [comment]
Posted my comment in https://github.com/getsentry/sentry-electron/issues/249
Does it ever gets called in case of a crash in main process?

**Pull request merged**